src/python/bot/tasks/impact_task.py

```diff
@@ -236,9 +244,24 @@ def get_impacts_on_prod_builds(testcase, testcase_file_path):
     # TODO(yuanjunh): undo the exception bypass for ES.
     logs.log_warn('Caught errors in getting impact on extended stable: %s' % e)
 
+  # Always record the affected head version.
+  start_revision, end_revision = get_start_and_end_revision(
+      testcase.regression, testcase.job_type)
+  build_revision_mappings = revisions.get_build_to_revision_mappings(None)
```

Comment:
no need to pass `None` explicitly here since that's the default.

Reply:
Done - thanks.